package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ceros-plugins",
+  "version": "1.0.0",
+  "description": "A collection of plugins and integrations for Ceros experiences",
+  "main": "Gruntfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
It probably isn't necessary to specify this, shouldn't npm give you a similar error message if `test` isn't defined?